docs/csharp/misc/cs0819.md

```diff
@@ -1,35 +1,43 @@
 ---
 title: "Compiler Error CS0819"
-ms.date: 07/20/2015
+ms.date: 09/13/2019
 f1_keywords: 
   - "CS0819"
 helpviewer_keywords: 
   - "CS0819"
 ms.assetid: a5369e03-eb7d-4c88-b390-51304bd8d1ae
 ---
 # Compiler Error CS0819
-Implicitly typed locals cannot have multiple declarators.  
-  
- Multiple declarators are allowed in explicit type declarations, but not with implicitly typed variables.  
-  
-## To correct this error  
-  
-1. Declare and assign a value to each implicitly typed local variable on a separate line.  
-  
-## Example  
- The following code generates CS0819:  
+Implicitly-typed variables cannot have multiple declarators.
+
+ Multiple declarators are allowed in explicit type declarations, but not with implicitly typed variables.
+
+## To correct this error
+
+There are two options:
+1. If the variables are of the same type, use explicit declarations.
+1. Declare and assign a value to each implicitly typed local variable on a separate line.
+
+## Example 1
```

Comment:
I don't think we need to add a number here

```suggestion
## Example
```

Reply:
Oh yes, I was going to add another example but forgot.
I'll add WIP and maybe add the other example tomorrow.

Reply:
@mairaw, I've added the second example and removed WIP